src/app/pages/RoflAppDetailsPage/GridRow.tsx

```diff
@@ -26,17 +15,20 @@ export const GridRow: FC<GridRowProps> = ({ label, children, tooltip }) => {
 
   return (
     <>
-      <StyledGrid item xs={4} lg={5}>
-        {label}:
-        {tooltip && (
-          <Tooltip title={tooltip} placement="top">
-            <InfoIcon htmlColor={COLORS.brandDark} fontSize="small" />
-          </Tooltip>
-        )}
-      </StyledGrid>
-      <StyledGrid item xs={8} md={7}>
-        {children ? <strong>{children}</strong> : t('common.missing')}
-      </StyledGrid>
+      <div className="grid grid-cols-12 border-b border-gray-100 py-2 md:py-4 gap-3">
```

Comment:
This creates separate grid for each row. We need to keep grid in parent and `GridRow` components handles rows only.

I am not checking details paddings and borders but try sth like 
```
MetaDataCard.tsx
          <div className="grid grid-cols-6 md:grid-cols-2 gap-4">
            <GridRow label={t('rofl.roflName')}>{metadata['net.oasis.rofl.name']}</GridRow>
```
```
GridRow.tsx
<>
      <div className="col-span-2 md:col-span-1">
        ...
      </div>

      <div className="col-span-4 md:col-span-1">
        {children ? <strong>{children}</strong> : t('common.missing')}
      </div>
    </>
```

Whatever solution you choose please align Policy section as well